dask/values.yaml

```diff
@@ -51,11 +52,12 @@ worker:
     pullSecrets:
     #  - name: regcred
   replicas: 3
-  aptPackages: >-
   default_resources:  # overwritten by resource limits if they exist
     cpu: 1
     memory: "4GiB"
   env:
+  #  - name: EXTRA_APT_PACKAGES
+  #    value: build-essential
```

Comment:
please put an example with multiple packages - I assume whitespace delimited is OK here

Reply:
whitespace delimited seems to work, I'm adding an example